HotSpot/DataManager.m

```diff
@@ -0,0 +1,36 @@
+//
+//  DataManager.m
+//  HotSpot
+//
+//  Created by drealin on 7/16/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import "DataManager.h"
+#import "Parse/Parse.h"
+
+@implementation DataManager
+
+# pragma mark - Class Methods
+
++ (void)configureParse {
+    
+    ParseClientConfiguration *config = [ParseClientConfiguration   configurationWithBlock:^(id<ParseMutableClientConfiguration> configuration) {
+        
+        configuration.applicationId = @"myAppId";
+        configuration.server = @"http://hotspot2017.herokuapp.com/parse";
+    }];
+    
+    [Parse initializeWithConfiguration:config];
+    
+}
+
++ (void)getHomeownersWithCompletion:(void(^)(NSArray<PFUser *> *homeowners, NSError *error))completion {
+    
+    PFQuery *query = [PFUser query];
+    [query whereKeyExists:@"address"]; // only homeowners have addresses
```

Comment:
"only homeowners have addresses"

I don't think this is a good assumption to make. In the future, anyone can have an address, and that would break this assumption. You should have a flag in the database suggesting that they are signed up to be a "homeowner" or "host" (whatever you choose to call it).